proposals/multiple-animations/usdskel.md

```diff
@@ -0,0 +1,140 @@
+# Multiple Animation in USDSkel
+
+This proposal outlines a way to add the ability for a Usd Skeleton to have multiple animation sources associated with it. 
+
+A follow up proposal will describe a possible way to add multiple animations to regular prims. I've chosen to split them out so as to focus discussion.
+
+This would be useful for runtimes like game engines or crowd simulation software where the ability to have access to multiple animations at once are useful so that the runtime can activate and blend between them as needed based on application specific context.
+
+It would also help bring better parity to the takes system in FBX, which has facilities for specifying multiple animations per object.
+
+Additionally, there is a desire to have no default animation as well, to speed up time to first load, since animation processing can be elided if the user isn’t using animations till a later time. e.g in Unreal, you may place a static actor for layout purposes, but only apply animation during runtime
+
+
+## Why not Variants? 
+
+Variants have a runtime cost associated with switching, and only expose a single variant at a time on the composed stage.
+Meanwhile, runtimes like game engines require access to the full range of options available and also switch often, such that recomposition is too heavy a cost to pay.
+
+## UsdSkel Schema Change Proposal
+
+USD Skeletons are unique in that they store animation externally to the prim itself. They are also the most likely candidate in a real time pipeline to have multiple animations, and game studios often will treat even basic transforms as a single joint skinned object.
+
+While the skeleton proposal wouldn’t allow generalization over non-skeletal animation such as transforming objects, it would at least cover the most common use cases requested in a way that should be fairly easy to provide quickly.
+This would also allow easier sharing of animations between multiple characters that share a skeleton.
+
+A future proposal would describe how we may add multiple animations to generic prims, and how that could play well with this Skeleton proposal.
+
+### Proposal 1
+
+In this version, we allow `skel:animationSource` to be a list, much like `blendShapeTargets` is. Since rels are allowed to be lists, this doesn’t change the schema.
+
+The API’s for the schema would need to be modified, with an overload for a Vec, and modify the getter/setter to take the first item in the list as the default active animation.
```

Comment:
In Proposal 1, it would be the first item in the list, so yeah we'd need something else if you wanted to activate a different item.

In Proposal 2, it would be whatever is set as the animationSource as is the case today. The animationLibrary would just be a list of associated items, whereas animation source would be the active item. It could be any of the items in animationLibrary and since there's no validation, it could also point to something not in the library.